LORIS_indepth.html

```diff
@@ -670,8 +402,213 @@ <h2>LORIS - Github <font color="yellow"> - Open Source</font></h2>
+		<td width="45%" style="vertical-align: middle">
+		  <a href="http://www.sciencedirect.com/science/article/pii/S105381190500710X?via%3Dihub" target="_blank"><img src="images/NIHPD_paper.png"></a>
+		</td>
+		
+                <td width="30%" style="vertical-align: middle">
+		  <img src="images/MRI_images_several.jpg">
+		</td>
+		<td width="30%" style="vertical-align: middle">
+		  <p><small>
+		      <ul>
+			<li>8 year study</li>
+			<li>Longitudinal data</li>
+			<li>532 subjects</li>
+			<li>8000 distinct variables</li>
+			<li>37000 individual assessments</li>
+			<li>T1, T2, PD, DTI</li>
+			<li>Spectroscopy</li>
+			<li>∼3TB of imaging data</li>
+			<li>2000 MRI acquisitions</li>
+		      </ul>
+		  </small></p>
+		</td>
+	    </tr></table>
+	  </section>
+
+<section data-transition="convex">
+            <a href="http://www.ibis-network.org/" target="_blank"><img src="images/IBIS.png"></a>
+            <table class="reveal">
+              <tr>
+                <td width="45%" style="vertical-align: middle">
+		  <a href="https://ajp.psychiatryonline.org/doi/abs/10.1176/appi.ajp.2011.11091447?url_ver=Z39.88-2003&rfr_id=ori%3Arid%3Acrossref.org&rfr_dat=cr_pub%3Dpubmed" target="_blank"><img src="images/IBIS_paper.png"></a>
+                </td>
+		
+                <td width="30%" style="vertical-align: middle">
+		  <img src="images/IBIS_image.png">
+                </td>
+                <td width="30%" style="vertical-align: middle">
+		  <p><small>
+                      <ul>
+                        <li>Infant brain development in autism</li>
+                        <li>3000+ scans</li>
+                        <li>700+ subjects</li>
+                        <li>7000 distinct variables</li>
+                        <li>150,000+ individual assessments</li>
+                        <li>T1, T2, DTI, BOLD</li>
+                        <li>∼5 TBs of imaging data</li>
+                        <li>Genetic/biospecimen data</li>
+		      </ul>
+		  </small></p>
+                </td>
+            </tr></table>
+	  </section>
+
+<!--CCNA-->
+	  <section data-transition="convex">
+            <a href="http://ccna-ccnv.ca/en/" target="_blank"><img src="images/CCNA.png"></a>
+	  </section>
+                  
+<!-- Big Brain -->
+        
+    <section>
+               <table class="reveal">
+                 <a href="Big_brain.html"><h2>BigBrain</h2></a>
+                 <tr>
+                   <td width="56%" style="vertical-align: middle">
+                     <a href="https://bigbrain.loris.ca/main.php" target="_blank"><img src="images/BigBrain.png"></a>
+                   </td>
+                   <td width="44%" style="vertical-align: middle">
+                     <a href="http://science.sciencemag.org/content/340/6139/1472" target="_blank"><img src="images/BigBrain_paper.png"></a>
+                   </td>
+               </tr></table>
+     	</section>
+                  
+<!-- OMEGA - small pilot, interest has spiked  -->
```

Comment:
remove...   

Replace with Open MNI iEEG atlas 
- [ ] to do: @christinerogers to provide content

Reply:
done